src/items/menu.js

```diff
@@ -109,6 +109,29 @@ const makeTree = (core, __, metadata) => {
  */
 export default class MenuPanelItem extends PanelItem {
 
+  init(state, actions) {
+    const onkeydown = ev => {
+      const checkKeys = (this.options.boundKey || 'Alt+a').toLowerCase().split('+');
+      const modifierNames =  ['ctrl', 'shift', 'alt', 'meta'];
+      const keyName = String(ev.key).toLowerCase();
+      const validKeypress = checkKeys.every(k => modifierNames.indexOf(k) !== -1
+        ? ev[k + 'Key']
+        : keyName === k);
+
+      if (!validKeypress) {
+        return;
+      }
+
+      this.panel.$element.querySelector('.osjs-panelitem-menu').click();
```

Comment:
This doesn't actually work because the className is invalid. Also, you don't have to add a new className for this, since the `Panel` class will add a custom attribute. You can use:

```
      this.panel.$element
        .querySelector('.osjs-panel-item[data-name=menu] > div')
        .click();
```

Reply:
I just now realize that this won't work properly if you have multiple menu buttons, but I can add a `querySelector` method (or something) on `PanelItem` class itself down the line to solve this.

Reply:
Well... there probably should only be one instance of this specific panelitem allowed anyways, but this will be on my mind bothering until I solve this edge case, lol.